dev/tasks/linux-packages/apache-arrow-release/yum/centos-7/Dockerfile

```
@@ -19,6 +19,14 @@ FROM centos:7
 
 ARG DEBUG
 
+# GH-42128
+# Switch repos to point to to vault.centos.org because Centos Stream 8 is EOL
+RUN sed -i \
+  -e 's/^mirrorlist/#mirrorlist/' \
+  -e 's/^#baseurl/baseurl/' \
+  -e 's/mirror\\.centos\\.org/vault.centos.org/' \
```

Ah, sorry. It seems that we can use `\` here.

```suggestion
  -e 's/mirror\.centos\.org/vault.centos.org/' \
```